benchmark/kernels/rotary_embedding/benchmark_mrope.py

```diff
@@ -0,0 +1,252 @@
+# Adapted from vLLM benchmark_mrope.py
```

Comment:
Remove it to sgl-kernel benchmark and get ci runner to avoid dead benchmark code.